src/core/json/jsont.rs

```diff
@@ -0,0 +1,102 @@
+#![allow(unused)]
+
+use std::collections::HashMap;
+
+pub trait JsonT {
+    type Output;
+    type Input;
```

Comment:
Use a type called `Json` for both `Input` and `Output`.

Reply:
I am thinking of using `JsonT` directly

Reply:
`type Output: JsonT;`